### DISCUSSION TITLE
There is a wait called "staleness_of" in selenium how to do the same in sbase

### DISCUSSION THREAD

**Discussion by elnatan2604** [seleniumbase/SeleniumBase#2147] [Q&A] [answered]
WebDriverWait(driver, 20).until(EC.staleness_of(
            driver.find_element(By.CSS_SELECTOR, "LOCATOR"))) -> I had this line in my code and I want to write it using sbase but my efforts to do so were unsuccessful.

In the end what I want is to my code to wait until an element is no longer attached to the DOM/ or to wait until my browser will finish to refresh itself

**Comment by mdmintz** [comment, marked as answer]
Use `self.wait_for_element_absent(selector, by="css selector", timeout=20)` to wait for an element to be gone from the DOM.

**Reply by elnatan2604** [reply]
Thank you :)
